multisig-factory/Cargo.toml

```diff
@@ -0,0 +1,21 @@
+[package]
+name = "multisig-factory"
+version = "0.1.0"
+authors = ["Near Inc <hello@near.org>"]
+edition = "2018"
+publish = false
+
+[lib]
+crate-type = ["cdylib", "rlib"]
```

Comment:
We can remove `rlib` for an easy reduction to code size:
https://www.near-sdk.io/reducing-contract-size/examples#removing-rlib-from-the-manifest

Reply:
factory side doesn't matter much. and removing this then stops from working the integration tests.